recipes/sipros/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sipros" %}
+{% set version = "4.01" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/thepanlab/Sipros4/releases/download/{{ version }}/siprosRelease.zip
+  sha256: 10030671b94a8a2ab354396178f35c30a8e8e0c633864533690c1ff285906c9c
+
+build:
+  noarch: generic
+  number: 0
+  skip: False
+  run_exports:
+    - {{ pin_subpackage('sipros', max_pin="x") }}
+    
+requirements:
+  run:
+    - bioconductor-biostrings
+    - mono
+    - numpy =1.16
+    - python =2.7
+    - r-base
+    - r-stringr
+    - r-tidyr
+    - scikit-learn =0.20.3
+
+test:
+  commands:
+    - configGenerator -h
+    - SiprosEnsembleOMP --help
+    - SiprosV4OMP --help
+    - Raxport -h
+    - python $CONDA_PREFIX/EnsembleScripts/sipros_psm_tabulating.py -v
+    - python $CONDA_PREFIX/V4Scripts/sipros_peptides_filtering.py -v
```

Comment:
You could symlink these two python scripts to `$PREFIX/bin`, or maybe use https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points?

Reply:
I added $PREFIX to differentiate the scripts from EnsembleScripts and V4Scripts, allowing them to be used like this: python $CONDA_PREFIX/EnsembleScripts/sipros_psm_tabulating.py -i input -c $CONDA_PREFIX/configTemplates/SiprosEnsembleConfig.cfg -o output. Is there any way to avoid using $CONDA_PREFIX by entry-points?